### PR TITLE
file proto

### PR DESCRIPTION
**Pull request by povolev15**
createed a protobuf for the file service

Related issue(s):
#265

### CODE REVIEW

File: services/state/file/file.proto [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "basic_types.proto";
+import "timestamp.proto";
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.consensus">>> This comment is special code for setting PBJ Compiler java package
```
Comment (iwsimon) [inline services/state/file/file.proto:29]: replace with com.hedera.hapi.node.state.file

**Review by iwsimon — APPROVED** [review]
LGTM

**Review by mhess-swl — APPROVED** [review]

**Review by iwsimon — APPROVED** [review]
LGTM

**Review by jasperpotts — APPROVED** [review]
Looks good

**Pull request merged**